src/main/starlark/builtins_bzl/common/cc/attrs.bzl

```diff
@@ -119,10 +119,36 @@ most build rules</a>.
    and that rule's <code>outs</code> are automatically added to
    this <code>cc_library</code>'s data files.
 </p>
-<p>Your C++ code can access these data files like so:</p>
+<p>Using the functionality defined by the <code>runfiles_src.h</code> located under 
+   <code>tools/cpp/runfiles/runfiles_src.h</code> Your C++ code can access these data files,
+   like so:
+</p>
 <pre><code class="lang-starlark">
-  const std::string path = devtools_build::GetDataDependencyFilepath(
-      "my/test/data/file");
+  
+  #include "runfiles_src.h"
```

Comment:
Is this actually the correct include spelling? I ended up having to include `"#include "tools/cpp/runfiles/runfiles.h"`", but I was not sure which is correct due to Bazel's lack of docs.

Reply:
@davidben That's the correct path, good catch.

Reply:
I had actually copied the header file into my project for testing directly, so this checks out